packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -0,0 +1,372 @@
+import { BigNumber, ERC20BridgeSource, SignedOrder } from '../..';
+import { constants } from '../../constants';
+
+import { BatchedOperation, DexSample } from './types';
+
+/**
+ * Composable operations that can be batched in a single transaction,
+ * for use with `DexOrderSampler.executeAsync()`.
+ */
+export const samplerOperations = {
```

Comment:
Are the contents of this the same as above, just refactored?

Reply:
that's correct, the file was becoming too big, so I factored out `samplerOperations`